### ISSUE TITLE
Issue of close accuracies 

### ISSUE THREAD

**Issue by boscy** [RUG-DeepLearning/DL-2020#5]
I was asked to post my email conversation here.   So here is the contact I had about this issue.

Dear TA's,

In doing assignment 1 for deep learning, we decided to use the MNIST dataset. We tried some configurations of settings, and got to the following results (Dropout 1 means we used a dropout layer, 0 means we did not):

![image](https://user-images.githubusercontent.com/35227051/74485252-9e017280-4eba-11ea-8209-76342c3d792b.png)

Now we were wondering whether these accuracies are enough apart from each other to write the report about, since they practically all lie between 95% and 98%. Does this matter for the report or should we aim for methods in which the accuracies lie further apart? If so, do you have any tips for us to achieve this?
Or is it mostly important that we explain the different methods correctly in the report, regardless of the accuracies?

______________________________________________________________________________

> I had a couple of questions regarding the models implemented:
> 1. Could you specify the architecture you used?
> 2. Could you also be a little more specific about the term "accuracy", is it training accuracy or validation accuracy?
> P.S. If you don't mind, could you raise this as an issue on the GitHub page we shared? It would help others who might be facing the same problems to see a similar question on the page.

__________________________________________________________________________________

1. We used a simple CNN, with use of tensorflow and keras. 
2. The accuracies in the table represent the accuracies for the testing phase (consisting of 1/7 of the total data set) , which is done after the training phase for 10 epochs (6/7 of the dataset). We also have plots for the training and validation loss and accuracy for each model, per epoch. 

__________________________________________________________________________________

> 1. I am assuming it is a simple CNN (none of the standardized architectures like VGG, ResNet, AlexNet etc.)
> 2. Correct me if I am wrong, I am assuming the dataset was downloaded from Yann LeCun's website (containing the labels for the test images as well). It is possible for a well tuned CNN to have very high accuracies with the dataset. You could test it by throwing some noise into your test images (using a data pipeline, add some Gaussian noise) and check if the accuracies are affected.
> 
> For the purpose of comparison, you could discuss why some of models exhibit slightly better accuracies and certain others have more errors. You could also try a standardized architecture and compare it's output against your network, with a subset of the parameters you tested.

__________________________________________________________________________________
1. It's indeed a simple CNN (I assume it is not one of those standardized architectures by default)
2. Actually, we download the dataset by the command ' tf.keras.datasets.mnist.load_data()'. We now indeed plan to add some noise to the test data. Otherwise we could always indeed compare the network to an architecture.

Thanks for the tips, I'll post our conversation on GitHub.
